packages/trpc/server/routers/viewer/bookings/confirm.handler.ts

```diff
@@ -304,6 +306,30 @@ export const confirmHandler = async ({ ctx, input }: ConfirmOptions) => {
     }
 
     await sendDeclinedEmails(evt);
+    // send BOOKING_REJECTED webhooks
+    const subscriberOptions = {
+      userId: booking.userId || 0,
+      eventTypeId: booking.eventTypeId || 0,
+      triggerEvent: WebhookTriggerEvents.BOOKING_REJECTED,
```

Comment:
```suggestion
      userId: booking.userId,
      eventTypeId: booking.eventTypeId,
      triggerEvent: WebhookTriggerEvents.BOOKING_REJECTED,
      teamId: booking.eventType?.teamId,
```
we are handling the 0 values now in getWebhooks and we need the teamId so team webhooks work